2022-02-04 03:24:59,362 INFO    MainThread:16080 [wandb_setup.py:_flush():71] setting env: {}
2022-02-04 03:24:59,362 INFO    MainThread:16080 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-04 03:24:59,363 INFO    MainThread:16080 [wandb_init.py:_log_setup():371] Logging user logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_032459-32fl5e7i\logs\debug.log
2022-02-04 03:24:59,363 INFO    MainThread:16080 [wandb_init.py:_log_setup():372] Logging internal logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_032459-32fl5e7i\logs\debug-internal.log
2022-02-04 03:24:59,363 INFO    MainThread:16080 [wandb_init.py:init():404] calling init triggers
2022-02-04 03:24:59,363 INFO    MainThread:16080 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-04 03:24:59,364 INFO    MainThread:16080 [wandb_init.py:init():460] starting backend
2022-02-04 03:24:59,364 INFO    MainThread:16080 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-02-04 03:24:59,376 INFO    MainThread:16080 [backend.py:ensure_launched():216] starting backend process...
2022-02-04 03:24:59,493 INFO    MainThread:16080 [backend.py:ensure_launched():222] started backend process with pid: 19628
2022-02-04 03:24:59,494 INFO    MainThread:16080 [wandb_init.py:init():469] backend started and connected
2022-02-04 03:24:59,495 INFO    MainThread:16080 [wandb_init.py:init():533] updated telemetry
2022-02-04 03:24:59,580 INFO    MainThread:16080 [wandb_init.py:init():563] communicating current version
2022-02-04 03:25:02,244 INFO    MainThread:16080 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-04 03:25:02,245 INFO    MainThread:16080 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-04 03:25:02,649 INFO    MainThread:16080 [wandb_init.py:init():606] starting run threads in backend
2022-02-04 03:25:05,694 INFO    MainThread:16080 [wandb_run.py:_console_start():1810] atexit reg
2022-02-04 03:25:05,695 INFO    MainThread:16080 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-04 03:25:05,695 INFO    MainThread:16080 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-04 03:25:05,707 INFO    MainThread:16080 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-04 03:25:05,707 INFO    MainThread:16080 [wandb_init.py:init():633] run started, returning control to user process
2022-02-04 03:25:06,395 INFO    MainThread:16080 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-04 03:25:06,397 INFO    MainThread:16080 [wandb_run.py:_restore():1752] restore
2022-02-04 03:25:06,751 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:25:07,238 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:25:07,738 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:25:07,854 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:25:07,965 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:25:08,082 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:25:08,187 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:25:08,299 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:08,411 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:08,521 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:08,633 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:08,744 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:09,565 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:25:10,352 INFO    MainThread:16080 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}
local_info {
}

2022-02-04 03:25:11,588 INFO    MainThread:16080 [wandb_run.py:_append_files():2180] logging synced files
